const asyncWrapper = (fn) => {
  return async (req, res, next) => {
    try {
      await fn(req, res, next);
    } catch (error) {
      next(error);
    }
  };
};
module.exports = asyncWrapper;

// di konapo ginamit tong wrapper. I choose the mnaula try catch instead
